client/my-sites/post-share/connection.jsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import FormToggle from 'components/forms/form-toggle/compact';
+import classNames from 'classnames';
+import SocialLogo from 'social-logos';
+
+const PostShareConnection = ( {
+	connection,
+	isActive,
+	onToggle,
+} ) => {
+	const {
+		external_display,
```

Comment:
minor: can we camelCase these?
``
const {
   externalDispaly: external_display,
  ...
} = connetion;
```